docker/dev/build/docker-entrypoint.sh

```diff
@@ -71,6 +71,42 @@ if [ "$1" = 'timesketch' ]; then
   # Add web user
   tsctl create-user --password "${TIMESKETCH_USER}" "${TIMESKETCH_USER}"
 
+  # Add Sigma rules
+  git clone https://github.com/SigmaHQ/sigma /usr/local/src/sigma
```

Comment:
I opt to create a file of the filenames that we want to import and loop over them while doing an os.walk. This way we have a text file that we can easily edit. We should also consider making this part of the release image as well.

Reply:
This would be a little larger effort to get done, not sure if you would be okay to still accept this here, if so: https://github.com/google/timesketch/issues/2415 to track it to be rewritten

Reply:
moving the paths to a file should be simple. Then we can store and update that in the data directory instead of hardcoding this in code.